### PR TITLE
Request to add fix in storage architecture

### PR DESCRIPTION
**Pull request by ghost**
- Storage putItem and getItem methods now return boolean
- Fix in UML diagram

### CODE REVIEW

File: src/main/resources/diagram_git_starship.puml [hunk 1]
```diff
@@ -15,8 +15,8 @@ package rape.brutal.gitstarship {
         -HashMap<String, IStorageable> itemsHashMap;
         -int capacity;
         +Set<String> getAllItemNames();
-        +void putItem(IStorageable iStorageable);
-        +void getItem(String itemName);
+        +boolean putItem(IStorageable iStorageable);
+        +boolean getItem(String itemName);
```
Comment (ViRGiL175) [inline src/main/resources/diagram_git_starship.puml]: Wrong return type.

File: src/main/java/rape/brutal/gitstarship/Storage.java [hunk 2]
```diff
@@ -36,11 +36,16 @@ public void setCapacity(int capacity) {
...
+    public boolean putItem(IStorageable iStorageable) {
 
+        if ((capacity - iStorageable.getSize()) >= 0) {
+            capacity -= iStorageable.getSize();
+            return true;
+        } else return false;
     }
 
-    public void getItem(String itemName) {
+    public boolean getItem(String itemName) {
```
Comment (ViRGiL175) [inline src/main/java/rape/brutal/gitstarship/Storage.java]: Wrong return type

**Review by ViRGiL175 — CHANGES_REQUESTED** [review]

**Comment by ViRGiL175** [comment]
Reviewed.

File: src/main/resources/diagram_git_starship.puml (hunk 1 above)
Comment (ghost) [reply src/main/resources/diagram_git_starship.puml]: Fixed?!

File: src/main/java/rape/brutal/gitstarship/Storage.java (hunk 2 above)
Comment (ghost) [reply src/main/java/rape/brutal/gitstarship/Storage.java]: Fixed?!

File: src/main/java/rape/brutal/gitstarship/Storage.java [hunk 3]
```diff
@@ -36,11 +36,16 @@ public void setCapacity(int capacity) {
         return itemsHashMap.keySet();
     }
 
-    public void putItem(IStorageable iStorageable) {
+    public boolean putItem(IStorageable iStorageable) {
 
+        if ((capacity - iStorageable.getSize()) >= 0) {
```
Comment (ViRGiL175) [inline src/main/java/rape/brutal/gitstarship/Storage.java]: It is only architecture fix, not feature fix. There is should be no realization.

**Review by ViRGiL175 — CHANGES_REQUESTED** [review]

**Comment by ViRGiL175** [comment]
Reviewed.

File: src/main/java/rape/brutal/gitstarship/Storage.java (hunk 3 above)
Comment (ghost) [reply src/main/java/rape/brutal/gitstarship/Storage.java]: Ok.

**Review by ViRGiL175 — APPROVED** [review]

**Comment by ViRGiL175** [comment]
Approved.

**Pull request merged**